### ISSUE TITLE
R not working

### ISSUE THREAD

**Issue by SarahWeber1699** [CrumpLab/psyc3400#3]
Hi professor,
I spoke to you today (Thursday) about R not working properly, I showed you on my laptop. Many of us are having this issue. Some people can’t even open R. You asked me to remind you to look into it. Please let us know what to do to fix it. 
Thanks!

**Comment by SarahWeber1699** [comment]
Hi professor,
I am using Rcloud because I couldn’t figure out how to actually install R and the lab professor told me it’s the same. I tried running them again and had the same issue. I attached a picture of the errors.

**Comment by CrumpLab** [comment]
Ok thanks, although the pictures didn't come through. That's ok thought, because I've seen some examples of the white screen problem.

I have been able to get R studio cloud to work fine for the labs. Have you been able to get R studio cloud working when you have a good internet connection? I'm wondering if the r studio cloud issue we were looking at in class was a matter of the wifi connection being a little bit spotty.

**Comment by SarahWeber1699** [comment]
Yes I have a strong internet connections and R studio cloud is running just the codes won’t run through. I keep getting error. I did figure out how to download R onto my computer however when I try to install the packages I need it said there’s an error and the file can’t be opened so I’m not really sure what the issue is.

**Comment by CrumpLab** [comment]
Thanks. Let's try two things with R-studio cloud. 

1. Download your .rmd file from R-studio cloud that is not working, and email it to me. I'll see if I can spot any issues.

2. I have been able to run all of lab 1 on R-studio cloud, so the issue is very puzzling. It's possible something happened to your project, you might try making a new project and seeing if it works in the new project

**Comment by CrumpLab** [comment]
3. I've found that some antivirus software on Windows may cause issues the prevent R-studio from accessing R. To test this, you can temporarily turn off your antivirus software, and then see if R-studio starts working.

**Comment by SarahWeber1699** [comment]
Hi professor,
Would you have time to meet on Tuesday in person to try and find out what
the problem is?

**Comment by SarahWeber1699** [comment]
Hi professor,
Can we do a little after? Like at 1:00?
